packages/devtools_app/lib/src/analytics/analytics.dart

```diff
@@ -239,18 +280,20 @@ void select(
 
 // Used only for Timeline Frame selection.
 void selectFrame(
-  String screenName,
-  String selectedItem, [
-  int rasterDuration, // Custom metric
-  int uiDuration, // Custom metric
-]) {
+  String screenName, {
+  @required Duration rasterDuration, // Custom metric
```

Comment:
I would omit these custom metric comments. not clear what they are communicating.